app/screens/node/Node.tsx

```diff
@@ -412,27 +446,35 @@ const Node = ({ history, location }: Props) => {
             style={{ marginRight: 15 }}
             imgPosition="before"
             width={180}
+            isDisabled={postSetupState === PostSetupState.STATE_NOT_STARTED}
           />
-          {postSetupState === PostSetupState.STATE_IN_PROGRESS && (
-            <Button
-              onClick={handlePauseSmeshing}
-              text="PAUSE POST DATA GENERATION"
-              img={pauseIcon}
-              isPrimary={false}
-              width={280}
-              imgPosition="before"
-            />
-          )}
-          {isPausedSmeshing && (
-            <Button
-              onClick={handleResumeSmeshing}
-              text="RESUME POST DATA GENERATION"
-              img={playIcon}
-              isPrimary
-              width={280}
-              imgPosition="before"
-            />
-          )}
+          {!isPausedSmeshing &&
+            [PostSetupState.STATE_IN_PROGRESS].includes(postSetupState) && (
```

Comment:
```suggestion
            postSetupState === PostSetupState.STATE_IN_PROGRESS && (
```
The same, but simpler, right? :)